docs/03.API-EXAMPLE.md

```diff
@@ -551,6 +551,32 @@ Value of x is 12
 Value of x is 17
 ```
 
+## Changing the seed of pseudorandom generated numbers
+
+If you want to change the seed of Math.random() generated numbers, you have to initialize srand with a seed value.
```

Comment:
Better wording perhaps: "you have to initialize the seed value with srand."

Also: code (or function references) are typeset differently from the rest of the text: use backticks around Math.random() and srand.